includes/schemas/class-prompts.php

```diff
@@ -355,6 +360,61 @@ function( $item ) {
 						],
 					],
 				],
+				// Required user inputs when auto-generating a prompt. These will be shown as fields in the UI.
+				'user_inputs'       => [
+					'name'     => 'user_inputs',
+					'type'     => 'array',
+					'required' => false,
+					'default'  => [],
+					'items'    => [
+						'type'                 => 'object',
+						'additionalProperties' => false,
+						'properties'           => [
+							// Slug for the input. Must be unique per prompt.
+							'name'        => [
+								'type' => 'string',
+							],
+							// Data type for the input.
+							'type'        => [
+								'type' => 'string',
+							],
+							// Label for the input.
+							'label'       => [
+								'type' => 'string',
+							],
+							// Help text describing the input.
+							'description' => [
+								'type' => 'string',
+							],
+							// Whether the input is required to generate the prompt or if it can be generated with an empty value. If true, the default value will be used when the input is empty.
+							'required'    => [
+								'type' => 'boolean',
+							],
+							// Default value of the input. This will be populated by default in the UI.
+							'default'     => [
+								'type' => [
```

Comment:
From what I see in the doc, and in the defaults below, this is always a string, isn't it?

Up there we are using the value to replace placeholders in the content, so we expect a string.

In what cases `default` and `value` are not strings?

Reply:
Yeah, I wasn't sure if there would ever be a case in which we'd want to add a user input that's a checkbox or other type of field, so I left the types as mixed. However, we could decide now that this feature will only be used for string-based placeholder replacements, and lock it down to `string` only. What do you think?

Reply:
Let's start with what we have and evolve as needed.

If we need other type of inputs, we will also need to treat them differently when running the importer, so I guess we can leave it for later

Reply:
269e0fc sets the `type` of `default` to `string` only.